src/main/rx/cc2500_common.c

```diff
@@ -0,0 +1,229 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+
+#include "platform.h"
+
+#if defined(USE_RX_FRSKY_SPI) || defined(USE_RX_SFHSS_SPI)
+
+#include "common/maths.h"
+
+#include "pg/pg.h"
+#include "pg/pg_ids.h"
+#include "pg/rx.h"
+#include "pg/rx_spi.h"
+
+#include "drivers/rx/rx_cc2500.h"
+#include "drivers/io.h"
+#include "drivers/time.h"
+
+#include "fc/config.h"
+
+#include "rx/rx.h"
+#include "rx/rx_spi.h"
+
+#include "rx/cc2500_common.h"
+
+static IO_t gdoPin;
+static IO_t bindPin = DEFIO_IO(NONE);
+static IO_t cc2500LedPin;
+static bool bindRequested;
+
+static bool lastBindPinStatus;
+#if defined(USE_RX_CC2500_SPI_PA_LNA)
+static IO_t txEnPin;
+static IO_t rxLnaEnPin;
+static IO_t antSelPin;
+#endif
+static int16_t rssiDbm;
```

Comment:
This should respect `USE_RX_FRSKY_SPI_TELEMETRY`.

Reply:
Do you mean rssiDbm?
In that case I think that it is better to separate TELEMETRY and RSSI.
RSSI can be displayed with OSD.

Reply:
Ok, fair enough. But in this case the calls to `cc2500setRssiDbm()` should also not be guarded by `USE_RX_FRSKY_SPI_TELEMETRY`, or displaying RSSI becomes meaningless.